From 9fbf8347b6ebdfa4fdf9a2a031aabe4932a06e3f Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 2 Aug 2018 11:21:30 +0800
Subject: [PATCH] dts:t4240rdb: remove the interrupts property of rtc device
 node

The INT pin of DS1374 does NOT route to the MPIC.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t4240rdb.dts |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/t4240rdb.dts b/arch/powerpc/boot/dts/fsl/t4240rdb.dts
index aa35ede..dddaba7 100644
--- a/arch/powerpc/boot/dts/fsl/t4240rdb.dts
+++ b/arch/powerpc/boot/dts/fsl/t4240rdb.dts
@@ -176,7 +176,6 @@
 			rtc@68 {
 				compatible = "dallas,ds1374";
 				reg = <0x68>;
-				interrupts = <0x1 0x1 0 0>;
 			};
 			monitor@2f {
 				compatible = "w83793";
-- 
1.7.9.5

